2015/proposals/girdhar-yashasvi.md

```diff
@@ -0,0 +1,183 @@
+#Result-aggregation server for the installation-test scripts
+
+##Personal Details:
+
+* Name: Yashasvi Girdhar
+* Email: yash.girdhar@gmail.com
+* Personal Webpage : http://about.me/yashgirdhar
+* Github profile : https://github.com/itsyash
+* IRC nick (on freenode) : yash_
+* Telephone: +91-8143914414
+* Country of residence: India
+* Timezone: IST (GMT+5:30)
+* Primary language: English
+
+##Abstract:
+
+Software carpentry has installation scripts which are used by students to check if  they have correctly installed the required software for a workshop.
+
+The script basically performs a test for each software and outputs the result. In case of a failure, it prints the cause of failure and some diagnostic information on how to install the software correctly.
+Until now, we do not collect these results anywhere. But there are number of reasons why we should be doing that.
+
+First of all, it will help us figure out some important information, such as how much trouble the students are having installing some software or which packages need more attention.
+
+Some sample queries that the system would serve will look like:
+
+* how many tests failed in a particular workshop, on windows.
+* what fraction of failed tests were on OS x, for a specific time period.
+* for a particular workshop, find number of failed version tests for git.
```

Comment:
The Git folks use “Git” for the project itself, and only use `git` when explicitly referring to the binary.  See [2de9b7113 (Documentation: the name of the system is 'Git', not 'git', 2013-01-21)][1]

[1]: http://git.kernel.org/cgit/git/git.git/commit/?id=2de9b71138171dca7279db3b3fe67e868c76d921